### PR TITLE
Tests for App Components

### PR DESCRIPTION
**Pull request by Epaltechs**
### In this project, we work together in pair-programming to:
- Set up React Testing Library.
- Write unit tests for the files operate.js and calculate.js using Jest.
- Create unit tests for all React components.
- Use Jest snapshots to test the components.
- Use React Testing Library to simulate user interaction.

### CODE REVIEW

File: src/components/Calculator.test.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    total: '',
+    operation: '',
+    next: '0',
+  };
+  const buttonNames = ['AC', '+/-', '%', '÷', '7', '8', '9', 'x', '4', '5', '6', '-', '1', '2', '3', '+', '0', '.', '='];
+  const tree = renderer
+    .create(<Design obj={data} data={buttonNames} />)
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
```
Comment (TedLivist) [inline src/components/Calculator.test.js]: - [ ] Using the [React Testing Library](https://testing-library.com/docs/react-testing-library/intro/), kindly add more tests simulating user interactions like clicking to perform calculations.

  You can also refer to this [tutorial on RTL](https://www.robinwieruch.de/react-testing-library/)

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Epaltechs and @SiansandaTamara,

Good job so far!
✔️ Snapshot tests
✔️ Tests are passing
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SiansandaTamara** [comment]
-ADD more test results

**Review by phelian23 — APPROVED** [review]
Hi @Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**